modules/core/src/main/java/org/apache/ignite/internal/processors/rest/GridRestProcessor.java

```diff
@@ -1039,6 +1043,30 @@ private void startProtocol(GridRestProtocol proto) throws IgniteCheckedException
         X.println(">>>   handlersSize: " + handlers.size());
     }
 
+    /**
+     * Logs REST request processing result.
+     *
+     * @param req Received REST request.
+     * @param fut Result of REST request processing.
+     */
+    private void logRestRequestResult(GridRestRequest req, IgniteInternalFuture<GridRestResponse> fut) {
+        if (log == null || !log.isInfoEnabled())
+            return;
+
+        GridRestResponse resp = null;
+
+        IgniteCheckedException reqProcErr = null;
+
+        try {
+            resp = fut.get();
+        }
+        catch (IgniteCheckedException e) {
+            reqProcErr = e;
+        }
+
+        log.info(String.format("REST request handled [req=%s, resp=%s, err=%s].", req, resp, reqProcErr));
```

Comment:
Why do we need `info` level here? I think every parameter of every request should be on `debug` or even `trace` level.

Reply:
Done.